template.rb

```diff
@@ -190,4 +192,8 @@ def create_initial_migration
   run_with_clean_bundler_env 'bin/rails db:migrate'
 end
 
+def install_graphviz
+  run_with_clean_bundler_env('brew install graphviz')
```

Comment:
Shouldn't we check if brew is available?

Reply:
I can add a check for this, but I'm not sure if there is a way to use macOS for development without brew. 
Probably this would make more sense to check if somebody is using a linux OS.

Reply:
Good point. Probably checking the OS is enough :D